plsync/mlabconfig.py

```diff
@@ -172,17 +189,18 @@ def export_experiment_records_v4(output, sites, experiment, decoration=''):
 
 
 def export_experiment_records_v6(output, sites, experiment, decoration=''):
-    comment(output, '%s v6%s' % (experiment.dnsname(), (
-        ' decorated' if decoration else '')))
+    comment(output, '%s v6%s' % (experiment.dnsname(), (' decorated' if
+                                                        decoration else '')))
     for site in sites:
         # TODO: change site['nodes'] to a pre-sorted list type.
         for node in sorted(site['nodes'].values(), key=lambda n: n.hostname()):
             # TODO: remove sitenames (or exclude mlab4's).
             if (node.ipv6_is_enabled() and experiment.ipv6(node)):
                 write_aaaa_record(output, experiment.sitename(node, decoration),
                                   experiment.ipv6(node))
-                write_aaaa_record(output, experiment.recordname(
-                    node,decoration), experiment.ipv6(node))
+                write_aaaa_record(output, experiment.recordname(node,
+                                                                decoration),
+                                  experiment.ipv6(node))
```

Comment:
What Peter said.

I'm also not super excited about some of it's decisions, but having something else make those decisions is more helpful. We can also experiment later with other versions of `yapf` -- it's behavior may have changed.